src/components/widgets/exclude-objects/ExcludeObjects.vue

```diff
@@ -41,30 +41,9 @@ import { Icons } from '@/globals'
 
 @Component({})
 export default class ExcludeObjects extends Mixins(StateMixin) {
-  get outlineNames () {
+  get parts () {
     const parts = this.$store.getters['parts/getParts']
-    const names = []
-    for (const name in parts) {
-      if (parts[name].outline.length >= 3) {
-        names.push(name)
-      }
-    }
-    return names
-  }
-
-  get iconNames () {
-    const parts = this.$store.getters['parts/getParts']
-    const state = this.$store.getters['parts/getPrintState']
-    const names = []
-    for (const name in parts) {
-      const excluded = this.isPartExcluded(name)
-      if (parts[name].target &&
-        ((state === 'printing' || state === 'paused') ||
-        ((state === 'complete' || state === 'cancelled' || state === 'error') && excluded))) {
-        names.push(name)
-      }
-    }
-    return names
```

Comment:
From my testing this isn't actually needed and causes some weird behavior where outlines are shown but icons aren't. Looking at the state the target and outline always seems to be set.